src/main/pages/che-6/user-guide/version-control.adoc

```diff
@@ -67,124 +97,127 @@ CHE_OAUTH_GITHUB_TOKENURI=https://github.com/login/oauth/access_token
+Currently it is not possible for Che to use oAuth integration with GitLab. Although GitLab supports oAuth for clone operations, pushes are not supported. You can track this https://gitlab.com/gitlab-org/gitlab-ce/issues/18106[GitLab issue] in their issue management system.
 
-[id="built-in-pull-request-panel"]
-== Built-In Pull Request Panel
+[id="using-the-built-in-pull-request-panel"]
+== Using the built-in pull request panel
 
-Within Eclipse Che there is a pull request panel to simplify the creation of pull requests for GitHub, BitBucket or Microsoft VSTS (with git) repositories.
+The pull request panel within Eclipse Che simplifies creation of pull requests for GitHub, BitBucket, and Microsoft VSTS (with Git) repositories.
 
 image::git/pr_panel.png[]
 
-[id="committer-name-and-email"]
-== Committer Name and Email
+[id="setting-the-committer-name-and-email"]
+== Setting the committer name and email
 
-Committer name and email are set in `Profile > Preferences > Git > Committer`. Once set each commit will include this information.
+Committer name and email are set using the *Profile* > *Preferences* > *Git* > *Committer* menu options. After it is set, each commit includes this information.
 
-[id="git-workspace-clients"]
-== Git Workspace Clients
+[id="working-with-git-workspace-clients"]
+== Working with Git workspace clients
 
-After importing repository, you can perform the most common Git operations using interactive menus or as terminal commands. Terminal git commands require its own authentication setup, which means that keys generated in the IDE will work only when Git is used in the IDE menus. Git installed in a terminal is a different git system. You may generate keys in `~/.ssh` there as well.
+After importing a repository, you can perform Git operations using interactive menus or terminal commands. Terminal Git commands require an authentication setup. The keys generated in the IDE will work only when Git is used in the IDE menus. Git installed in a terminal is a different Git system. You may generate keys in the `~/.ssh` file.
 
 image::git/git.gif[]
 
-To faster access to the most frequently used git functionality one can use keyboard shortcuts:
+Use the following keyboard shortcuts to quickly access the most frequently used Git functionalities:
 
 * Commit: `Alt + C`
 * Push to remote: `Alt + Shift + C`
 * Pull from remote: `Alt + P`
 * Work with branches: `Ctrl + B`
 * Compare current changes with the latest repository version: `Ctrl + Alt + D`
 
-[id="git-in-project-tree-and-editor"]
-== Git in Project Tree and Editor
+[id="color-codes-for-files-in-the-project-explorer-and-editor"]
+== Color codes for files in the Project Explorer and Editor
```

Comment:
"Editor" is capitalized here, but is not capitalized within the text of this section.